pkg/remoteclient/remoteclient.go

```diff
@@ -198,6 +200,10 @@ const (
 func buildScheme() (*runtime.Scheme, error) {
 	scheme := runtime.NewScheme()
 
+	if err := capiv1.AddToScheme(scheme); err != nil {
```

Comment:
Is this actually needed for the remote client? I thought all the CAPI things were local.